docs/primer_short.md

```diff
@@ -0,0 +1,106 @@
+- Each individual has the possibility of becoming exposed $E$, infected $I$, and then seek and receive a test $S$, with conditional probabilities depending on vaccination status:
+  - E.g., the probability that a vaccinated person will receive a test is $P[E|V] \times P[I|E,V] \times P[S|I,V]$
+  - There is only one opportunity for exposure per individual
+  - Note that the $I \to S$ transition represents a combination of developing symptoms, seeking healthcare, and actually receiving a test
+- Every individual also has the opportunity to seek and receive a test for reasons unrelated to infection, while in uninfected status $X$
+  - E.g., the probability that a vaccinated person will seek a test like this is $P[S|X,V]$
+- People who receive tests are either positive ($P$) or negative ($N$)
+  - The test has sensitivity $p_\mathrm{sens}$ and specificity $p_\mathrm{spec}$, such that, e.g., $P[P|S,V] = p_\mathrm{sens}$
+
+#### Quantity of interest
+
+We are interested in protection against symptomatic disease, conditioned on exposure:
+
+```math
+\mathrm{VE}_{SP} = 1 - \frac{P[S | V, E]}{P[S | U, E]}
+```
+
+#### Measured quantities
+
+The expected values of the numbers of infected and uninfected, stratified by vaccination status, are:
+
+```math
+\begin{align*}
+\mathbb{E}[C_{VI}] &= n v \times P[E|V] \times P[I|E,V] \times P[S|I,V] \\
+\mathbb{E}[C_{VX}] &= n v \times P[S|X,V] \\
+\mathbb{E}[C_{UI}] &= n (1-v) \times P[E|U] \times P[I|E,U] \times P[S|I,U] \\
+\mathbb{E}[C_{UX}] &= n (1-v) \times P[S|X,U] \\
+\end{align*}
+```
+
+The actual values from the TND trial are affected by test performance:
+
+```math
+\begin{align*}
+\mathbb{E}[C_{VP}] &= p_\mathrm{sens} \mathbb{E}[C_{VI}] + (1 - p_\mathrm{spec}) \mathbb{E}[C_{VX}] \\
+\mathbb{E}[C_{VN}] &= (1 - p_\mathrm{sens}) \mathbb{E}[C_{VI}] + p_\mathrm{spec} \mathbb{E}[C_{VX}] \\
+\mathbb{E}[C_{UP}] &= p_\mathrm{sens} \mathbb{E}[C_{UI}] + (1 - p_\mathrm{spec}) \mathbb{E}[C_{UX}] \\
+\mathbb{E}[C_{UN}] &= (1 - p_\mathrm{sens}) \mathbb{E}[C_{UI}] + p_\mathrm{spec} \mathbb{E}[C_{UX}] \\
+\end{align*}
+```
+
+#### Odds ratio estimator
+
+An estimator of the desired quantity is:
+
+```math
+\hat{\mathrm{VE}} = 1 - \frac{C_{VP} C_{UN}}{C_{UP} C_{VN}}
+```
+
+When there is perfect test performance, the TND counts reduce to the actual disease status counts, e.g., $C_{VP} = C_{VI}$. In this case, the expected value of the estimator is:
+
+```math
+\mathbb{E}\left[\hat{\mathrm{VE}}\right] = 1 - \frac{P[E|V] \times P[I|E,V] \times P[S|I,V] \times P[S|X,U]}{P[E|U] \times P[I|E,U] \times P[S|I,U] \times P[S|X,V]}
+```
+
+If we further assume that:
+
+1. the probability of exposure is identical among the vaccinated and unvaccinated, i.e., $P[E|V] = P[E|U]$, and
+2. vaccination does not affect progression to testing among the uninfected, i.e., $P[S|X,U]=P[S|X,V]$
```

Comment:
The algebra works out! Great!

I will restate my case for separating disease-progression-given-infection (purely immunological) from test-seeking-given-progression (purely behavioral). Your mathematical formulation makes this easier to discuss now!

As it stands, I see two problems with your derivation:

1) The term P[S | I] includes both immunology and behavior (see line 42). Because both P[S | I,V] and P[S | I,U] play a role in the final simplified estimator, this means your VE_SP estimate incorporates behavioral differences between the I,V and the I,U populations. But I suspect most people would rather interpret VE_SP as the purely immunological effect of the vaccine, not incorporating the differential behavior of vaccinated vs. unvaccinated people.

2) You must assume that P[S | X,V] = P[S | X,U]. But this is probably not true. Those who choose to get vaccinated are probably also more likely to seek a test (regardless of whether they are actually infected, because people seek a test before they know the cause of their respiratory distress).

So instead, say that P[S | I] = P[P | I] * P[S | P], where the P = progression to severe symptoms. Essentially, seeking a test given infection has been decomposed into two steps: progressing immunologically then seeking behaviorally. 

Now you have:
```math
\mathbb{E}\left[\hat{\mathrm{VE}}\right] = 1 - \frac{P[E|V] \times P[I|E,V] \times P[P|I,V] \times P[S|P,V] \times P[S|X,U]}{P[E|U] \times P[I|E,U] \times P[P|I,U] \times P[S|P,U] \times P[S|X,V]}
```
 
The first terms in the numerator and denominator cancel out because of your assumption of equal exposure. The final two terms in the numerator and denominator also cancel out, because we assume that P[S | P,V] = P[S | X,V] and P[S | P,U] = P[S | X,U]. This is a reasonable assumption, because people don't know whether they are infected or not when they seek a test - all they know is that they are in respiratory distress, regardless of whether the cause is P or X. Vaccination status may affect their test-seeking behavior, but that doesn't matter - the cancellation still happens.

You are left with: 
```math
\mathbb{E}\left[\hat{\mathrm{VE}}\right] = 1 - \frac{P[I|E,V] \times P[P|I,V]}{P[I|E,U] \times P[P|I,U]} = 1 - \frac{P[P|E,V]}{P[P|E,U]}
```

This solves both problems above: 1) it estimates only the immunological effect of the vaccine in preventing infection and/or progression, and 2) it never requires assuming that vaccinated and unvaccinated people have identical test-seeking behavior.

Reply:
I think I finally understand what you've been saying. Let check my understanding.

There are two disease progressions (where I'm changing my notation again, I think for the last time):

- exposed $E$ to infected $I$ to infected symptomatic $S_I$ to tested $T$
- not infected $X$ to uninfected symptomatic $S_X$ to tested $T$

The expected value of the TND estimator is:

$$
1 - \frac{P[E|V] \times P[I|E,V] \times P[S_I|I,V] \times P[T|S_I,V] \times P[S_X|U] \times P[T|S_X,U]}{\text{same but with U and V swapped}}
$$

If we assume:

1. Equal exposure rates: $P[E|V] = P[E|U]$ ==> first term drops
2. Non-infection symptoms arise with the same probability among the vaccinated and unvaccinated: $P[S_X|V]=P[S_X|U]$ ==> term 5 cancels
3. Vaccinated people seek tests with some probability when they have symptoms, regardless of whether those symptoms are caused by infection or not: $P[T|S_I,V] = P[T|S_X,V]$, and ditto for unvaccinated people: $P[T|S_I,U] = P[T|S_X,U]$ ==> terms 4 & 6 drop

then this reduces to:

$$
1 - \frac{P[S_I|E,V]}{P[S_I|E,U]}
$$

which is $\mathrm{VE}_{SP}$ if we take "P" to be progression to symptoms.

I'm not sure why this took me so long. I think I needed to do it in my own notation?

Reply:
This is going to be a pain if you violate any of these assumptions, though, because they you need to separate input $P[S_I|I]$ and $P[T|S_I]$, and similarly $P[S_X]$ and $P[T|S_X]$, into a simulation like in the widget. I'll need to figure that out.

But at least we're on the same page now!

Reply:
Bingo! Your latest explanation looks exactly right to me!

Extra parameters can be a pain, but they can also be super interesting. For example, it seems intuitive that VE_SP will become biased if the transmission dynamics of the focal pathogen I differ from the dynamics of other causes of symptoms X. But interestingly, this bias only arises if vaccination lowers $P[I | E]$. On the other hand, if vaccination only lowers $P[S_I | I]$ without affecting  $P[I | E]$, then different dynamics aren't a problem, because the difference is identical between the V and U populations.

But we can worry about if/how such oddities should feature in the widget later. Glad we're reconciled!

Reply:
I'm going to break this out in #9